lib/util/test_work_area.cpp

```diff
@@ -147,14 +147,36 @@ static bool test_work_area_copy_parent__( const TestArea * work_area , const std
   }
 }
 
+static char *create_test_path(const std::string &test_name, const char *prefix,
```

Comment:
Standard C++ indentation in libecl is 2 spaces 🤷

Reply:
Also, would the function read nicer as a for-loop? Just I suggestion; I leave the decision to you ;)

```c++
static char *create_test_path(
    const std::string &test_name,
    const char *prefix,
    const char *user_name
) {
  const int MAX_TRIES = 10;
  for (int try_count = 0; try_count < MAX_TRIES; ++try_count) {
    unsigned int random_int;
    util_fread_dev_urandom(sizeof random_int, (char *) &random_int);
    random_int = random_int % 100000000;

    char *test_path = util_alloc_sprintf(
        TEST_PATH_FMT, user_name, test_name.c_str(), random_int
    );

    char *test_cwd = util_alloc_sprintf(FULL_PATH_FMT, prefix, test_path);
    free(test_path);

    if(util_mkdir_p(test_cwd))
      return test_cwd;
    free(test_cwd);
  }

  util_abort(
      "%s: failed to make test directory after %d tries: %s - aborting\n",
      __func__, MAX_TRIES, test_name.c_str()
  );
  return NULL;
}
```

Reply:
> Standard C++ indentation in libecl is 2 spaces 🤷
For the rest of the file, this is true, but not for the TestArea class.

Reply:
I took your subtle "hint" and implemented it your way. With some minor changes of course.

Reply:
It was not intended as a subtle "hint", but as an invitation to a discussion for which you are free to ignore. If you prefer the previous implementation I'm perfectly fine with merging that ;)

Reply:
No I liked your way better. Just wanted to show some attitude.

Reply:
I like it ;)